### PR TITLE
Add support for xsd:int and xsd:long

### PR DESCRIPTION
**Pull request by ebremer**
reference: https://github.com/filip26/titanium-json-ld/issues/178

### CODE REVIEW

**Comment by ebremer** [comment]
add support for xsd:int
https://github.com/filip26/titanium-json-ld/issues/178

File: src/main/java/com/apicatalog/jsonld/serialization/RdfToObject.java [hunk 1]
```diff
@@ -109,6 +109,10 @@ public JsonObject build() throws JsonLdError {
 
                     convertedValue = Json.createValue(Long.parseLong(literal.getValue()));
 
+                } else if (XsdConstants.INT.equals(literal.getDatatype())) {
```
Comment (filip26) [inline src/main/java/com/apicatalog/jsonld/serialization/RdfToObject.java:112]: What about to just extend line 108?
e.g.
```java
  } else if (XsdConstants.INTEGER.equals(literal.getDatatype()) || XsdConstants.INT.equals(literal.getDatatype())) {
```

**Comment by filip26** [comment]
@ebremer thank you for the PR! Would you mind to add a simple test case?  In this case a test in a form input/expected output is enough.

https://github.com/filip26/titanium-json-ld/blob/main/src/test/java/com/apicatalog/jsonld/CustomTest.java

reads this test manifest:

https://github.com/filip26/titanium-json-ld/blob/main/src/test/resources/com/apicatalog/jsonld/test/manifest.json

Check test `t0006` for an example.

**Comment by ebremer** [comment]
I'll add the test case.

**Comment by ebremer** [comment]
added the test and included support for longs too.

**Comment by filip26** [comment]
@ebremer Thank you! Please feel to open a new issue or PR.

**Pull request merged**